src/main/java/model/card/Cards.java

```diff
@@ -0,0 +1,72 @@
+package model.card;
+
+import static constant.BlackjackConstant.BUST_NUMBER;
+
+import java.util.ArrayDeque;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Deque;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class Cards {
+
+    private final List<Card> cards;
+
+    public Cards() {
+        this.cards = new ArrayList<>();
+    }
+
+    public void add(Card card) {
+        cards.add(card);
+    }
+
+    public int calculateNearestTotal() {
+        Set<Integer> candidates = new HashSet<>();
+        generateCandidates(candidates, totalWithoutAce(), aceCount());
```

Comment:
```suggestion
        Set<Integer> candidates =  generateCandidates(candidates, totalWithoutAce(), aceCount());
```
candidates를 넘겨주는 것이 아니라 값을 리턴받도록 하는건 어떨까요?

Reply:
아 네네! 가독성 측면이든, 책임 분리 측면이든 그 방식이 더 좋은 것 같아요. 😚